presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -186,12 +186,26 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kHttpsClientCertAndKeyPath{
       "https-client-cert-key-path"};
 
-  /// Number of threads for async io. Disabled if zero.
-  static constexpr std::string_view kNumIoThreads{"num-io-threads"};
-  static constexpr std::string_view kNumConnectorIoThreads{
-      "num-connector-io-threads"};
-  static constexpr std::string_view kNumQueryThreads{"num-query-threads"};
-  static constexpr std::string_view kNumSpillThreads{"num-spill-threads"};
+  /// Floating point number used in calculating how many threads we would use
+  /// for IO executor for connectors to do preload/prefetch:
+  /// hw_concurrency x multiplier.
+  /// If 0.0 then connector preload/prefetch is disabled.
+  /// 0.0 is default.
+  static constexpr std::string_view kConnectorPrefetchNumIoThreadsHwMultiplier{
```

Comment:
```
s/kConnectorPrefetchNumIoThreadsHwMultiplier/kConnectorNumIoThreadsHwMultiplier/
```
We don't need prefetch prefix

Reply:
Discussed offline. Will remove 'Prefetch'.